CONTRIBUTING.md

```diff
@@ -57,6 +57,16 @@ python3 -m http.server 8080
 
 If you want to help maintain translations in the future, add yourself to `.github/DOCS_ISSUE_TEMPLATE.md`.
 
+### Tips
+
+Here are a few tips when contributing to the documentation:
+- We use [docsify](https://docsify.js.org) for document rendering. Refer to the docsify documentation for more context/examples regarding the syntax, usage, and capabilities.
+- We prefer to reference real files in GitHub rather than creating snippets. (Some old examples make use of snippets but we are slowly migrating away.)
```

Comment:
```suggestion
- We prefer to reference real files in GitHub rather than creating snippets (some old examples make use of snippets but we are slowly migrating away).
```